http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20200123_cluster_table_gce_refactor.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="leonardo" author="qi" id="cluster_table_gce_refactor">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="RUNTIME_CONFIG"/>
+            </not>
+        </preConditions>
+
+        <dropNotNullConstraint columnName="masterMachineType" tableName="CLUSTER" columnDataType="varchar(256)"/>
+        <dropNotNullConstraint columnName="masterDiskSize" tableName="CLUSTER" columnDataType="INT"/>
+        <dropNotNullConstraint columnName="numberOfWorkers" tableName="CLUSTER" columnDataType="INT"/>
+
+<!--                  Create RUNTIME_CONFIG table   -->
+        <createTable tableName="RUNTIME_CONFIG">
+            <column name="id" type="BIGINT" autoIncrement="true">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="clusterId" type="BIGINT"></column>
+            <column name="cloudService" type="ENUM('DATAPROC', 'GCE')" defaultValue="DATAPROC">
+                <constraints nullable="false"/>
+            </column>
+
+            <column name="machineType" type="varchar(256)" defaultValue="n1-standard-4">
```

Comment:
Do we need defaults at the DB level? If the app fails to set these I'd kind of rather it'd fail than set some hidden default.

Reply:
this was probably copied over from previous changelog..I'll remove `default`